api/scripts/fill-in-assessment-method.js

```diff
@@ -0,0 +1,35 @@
+require('dotenv').config();
+
+const { knex } = require('../db/knex-database-connection');
+const logger = require('../lib/infrastructure/logger');
+
+async function fillInAssessmentMethod() {
+
+  const chunkSize = 50000;
+  const maxId = (await knex('assessments').max('id').first()).max;
+
+  for (let startId = 0; startId < maxId; startId += chunkSize) {
+    const result = await knex.raw(`
+      UPDATE "assessments" SET "method" =
+        CASE
+          when "type" = 'COMPETENCE_EVALUATION' then 'SMART_RANDOM'
+          when "type" = 'CAMPAIGN' then 'SMART_RANDOM'
+          when "type" = 'PLACEMENT' then 'SMART_RANDOM'
+          when "type" = 'CERTIFICATION' then 'CERTIFICATION_DETERMINED'
+          when "type" = 'DEMO' then 'COURSE_DETERMINED'
+          when "type" = 'PREVIEW' then 'CHOSEN'
+        END
+      WHERE ID BETWEEN ?? AND ??;
+    `, [startId, startId + chunkSize]);
```

Comment:
Suggestions : moins de `knex.raw`, ne pas mettre à jour quand une `method` a déjà été renseignée, et ne pas mettre à jour plus de lignes que souhaité (`BETWEEN` est inclusif) :

```suggestion
    const result = await knex('assessments')
      .whereBetween('id', [ startId, startId + chunkSize - 1 ])
      .whereNull('method')
      .update({
        method: knex.raw(`
	      CASE "type"
	        WHEN 'COMPETENCE_EVALUATION' THEN 'SMART_RANDOM'
	        WHEN 'CAMPAIGN' THEN 'SMART_RANDOM'
	        WHEN 'PLACEMENT' THEN 'SMART_RANDOM'
	        WHEN 'CERTIFICATION' THEN 'CERTIFICATION_DETERMINED'
	        WHEN 'DEMO' THEN 'COURSE_DETERMINED'
	        WHEN 'PREVIEW' THEN 'CHOSEN'
	      END`
	    });
```